src/Plugins/LevelDBStore/Plugins/Storage/Snapshot.cs

```diff
@@ -9,61 +9,48 @@
 // Redistribution and use in source and binary forms with or without
 // modifications are permitted.
 
-using Neo.IO.Data.LevelDB;
+using Neo.IO.Storage.LevelDB;
 using Neo.Persistence;
 using System.Collections.Generic;
-using LSnapshot = Neo.IO.Data.LevelDB.Snapshot;
 
 namespace Neo.Plugins.Storage
 {
     internal class Snapshot : ISnapshot
     {
-        private readonly DB db;
-        private readonly LSnapshot snapshot;
-        private readonly ReadOptions options;
-        private readonly WriteBatch batch;
+        private readonly DB _db;
+        private readonly SnapShot _snapshot;
+        private readonly WriteBatch _batch;
+        private readonly ReadOptions _readOptions;
 
         public Snapshot(DB db)
         {
-            this.db = db;
-            snapshot = db.GetSnapshot();
-            options = new ReadOptions { FillCache = false, Snapshot = snapshot };
-            batch = new WriteBatch();
+            _db = db;
+            _snapshot = db.CreateSnapshot();
+            _batch = new();
+            _readOptions = new ReadOptions { FillCache = false, Snapshot = _snapshot };
         }
 
-        public void Commit()
-        {
-            db.Write(WriteOptions.Default, batch);
-        }
+        public void Commit() =>
+            _db.Write(_batch);
 
-        public void Delete(byte[] key)
-        {
-            batch.Delete(key);
-        }
+        public void Delete(byte[] key) =>
+            _batch.Delete(key);
 
-        public void Dispose()
-        {
-            snapshot.Dispose();
-        }
+        public void Dispose() =>
+            _snapshot.Dispose();
 
-        public IEnumerable<(byte[] Key, byte[] Value)> Seek(byte[] prefix, SeekDirection direction = SeekDirection.Forward)
-        {
-            return db.Seek(options, prefix, direction, (k, v) => (k, v));
-        }
+        public void Put(byte[] key, byte[] value) =>
+            _batch.Put(key, value);
 
-        public void Put(byte[] key, byte[] value)
-        {
-            batch.Put(key, value);
-        }
+        public bool Contains(byte[] key) =>
+            _db.Contains(key, _readOptions);
 
-        public bool Contains(byte[] key)
-        {
-            return db.Contains(options, key);
-        }
+        public byte[] TryGet(byte[] key) =>
+            _db.Get(key, _readOptions);
 
-        public byte[] TryGet(byte[] key)
-        {
-            return db.Get(options, key);
-        }
+        public IEnumerable<(byte[] Key, byte[] Value)> Seek(byte[] prefix, SeekDirection direction = SeekDirection.Forward) =>
+            direction == SeekDirection.Forward
+                ? _db.Seek(prefix, new ReadOptions { FillCache = false, })
+                : _db.SeekPrev(prefix, new ReadOptions { FillCache = false, });
```

Comment:
```suggestion
                ? _db.Seek(prefix, ReadOptions.Default)
                : _db.SeekPrev(prefix, ReadOptions.Default);
```

Reply:
This will cause problem, because `ReadOptions.Default` is `static readonly` and will use the same instance. So if someone changes `ReadOptions.Default` parameters will cause problems for us. Thats why I coded it this way.

Reply:
`ReadOptions.Default` isnt the same as `new ReadOptions { FillCache = false, }`